### DISCUSSION TITLE
rcut while running the molecular dynamics simulations using trained potential

### DISCUSSION THREAD

**Discussion by rajnichahal** [deepmodeling/deepmd-kit#1902] [General]
Hi,
I was wondering if one could take a different rcut value when running the  molecular dynamics simulations using trained DeePMD potential. 
For example, say rcut=8 Angstrom when training the potential, is it possible to select an rcut=6 Angstrom when using the potential in LAMMPS?
Thanks!

**Comment by njzjz** [comment]
No, the potential is designed to be continuous at the rcut point.

**Reply by rajnichahal** [reply]
Thanks!
